### ISSUE TITLE
Cannot install from Nuget

### ISSUE THREAD

**Issue by arame** [AutoMapper/AutoMapper#804]
When I try to install Automapper from Nuget, I get this error;
The remote server returned an error: (403) Forbidden.

How do I fix this?

**Comment by jbogard** [comment]
What version of Visual Studio are you using? And does this happen with all NuGet packages or just AutoMapper? And what NuGet repository URL are you trying to hit?

**Comment by arame** [comment]
I am using VS Premium 2013 Version 12.0.31101.00 Update 4. I have only noticed this with Automapper, other Nuget packages are OK. 
The URL is https://www.nuget.org/packages/AutoMapper/3.3.1

**Comment by jbogard** [comment]
No sorry I meant what NuGet URL are you pointing to, not which package. This is in your NuGet config.

**Comment by arame** [comment]
https://www.nuget.org/api/v2/

**Comment by jbogard** [comment]
Hmmm, I just can't reproduce this one. Can you try the pre-release alpha version?

**Comment by arame** [comment]
Still same problem.
![automapperfail](https://cloud.githubusercontent.com/assets/688324/9032103/1d1eb20c-39b3-11e5-9ff3-cd2f037385b5.jpg)

**Comment by arame** [comment]
Put this on hold, I am looking at a firewall issue in relation to this which might be the cause. Waiting on colleagues to respond.

**Comment by arame** [comment]
Ok the problem was not with automapper. Recently the nugget service (https://visualstudiogallery.msdn.microsoft.com/services/dev12/extension.svc)  was blocked by company policy as a download site, and now it is unblocked the problem is solved. I incorrectly answered your previous question, I thought Nuget was working when it was not.

**Comment by bdaniel7** [comment]
I have a similar problem, trying to install Automapper 5.2.0 in VS 2013, from a lan share:
PM> Install-Package AutoMapper
Install-Package : Unable to find package 'AutoMapper'.

From VS 2015 I can see the nuget.

**Comment by jbogard** [comment]
Updated your NuGet client?

**Comment by bdaniel7** [comment]
Yes, just did that and now I can install it.
